tools/Customization/DevHome.Customization/ViewModels/FileExplorerViewModel.cs

```diff
@@ -292,4 +293,17 @@ public async void OnToggledRepositoryStatusSettingAsync(bool value)
 
         await LocalSettingsService!.SaveSettingAsync("ShowRepositoryStatus", value);
     }
+
+    private void AdjustFocus(object sender)
+    {
+        var addRepositoryCard = sender as SettingsCard;
+        if (addRepositoryCard != null)
+        {
+            addRepositoryCard.IsTabStop = true;

```

Comment:
Can we set IsTabStop on the SettingsCard in the xaml instead of here?

Reply:
I did initially set IsTabStop in the xaml, however, I found that the focus behavior was inconsistent. Debugging showed that IsTabStop was false for the settings card even when set to true in the xaml. I do not know why it appeared as false though I did find that setting IsTabStop in the function reliably helped Dev Home regain focus after File Explorer closes.